components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/java/org/wso2/carbon/idp/mgt/ui/util/IdPManagementUIUtil.java

```diff
@@ -559,6 +560,12 @@ private static void buildSCIMProvisioningConfiguration(IdentityProvider fedIdp,
             fedIdp.setDefaultProvisioningConnectorConfig(proConnector);
         }
 
+        if (paramMap.get("scim-version") != null) {
+            scimVersionProp = new Property();
+            scimVersionProp.setName("scim-version");
```

Comment:
shall we define a constant for "scim-version"?

Reply:
@emswbandara These commits had been added with [1] with some improvements. I hope it's better to review the PR [1] and give comments.

[1] https://github.com/wso2/carbon-identity-framework/pull/1493

Reply:
@nilasini does this mean that we can close this PR and track only the [1] ?

Reply:
This PR is improved with https://github.com/wso2/carbon-identity-framework/pull/1493